### DISCUSSION TITLE
"Reject" Status by Directus

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15125] [Archive]
Hi,

is it possible to make this scenario by Directus? What must be used for such scenarios? Flows?

- User A with role "WRITER" create Article A
- User B with role "WRITER" create Article B
- **if** user "EDITOR" set to Article A status "REJECTED" > **then** only User A, who have written article A, can see this status REJECTED

Best regards

**Comment by licitdev** [comment]
This can be achieved with Custom Read Permissions for the writer role. 
Ref: https://docs.directus.io/configuration/users-roles-permissions.html#configure-permissions

In the following example, a writer can see all articles that are not rejected, and their rejected articles.

![image](https://user-images.githubusercontent.com/26413686/185218089-55be94bd-58b6-4066-a095-fa0a92cca9e1.png)

An alternative is to simply allow a writer to only see their articles, so the status does not matter.

![image](https://user-images.githubusercontent.com/26413686/185218345-5d8414ab-0771-4b5b-b758-4e6fb4178d04.png)

You will have to select the `user_created` field when creating the `article` table to use the `$CURRENT_USER` dynamic variable. Ref: https://docs.directus.io/reference/filter-rules.html#dynamic-variables
